deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: istio-operator
       containers:
         - name: istio-operator
-          image: gcr.io/istio-testing/operator:master-latest-daily
+          image: gcr.io/howardjohn-istio/operator:master-latest-daily
```

Comment:
intentional?

Reply:
Nope, was testing to make sure this worked, good catch. reverted.